core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -380,10 +381,8 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     }
 
     // System property spark.yarn.app.id must be set if user code ran by AM on a YARN cluster
-    // yarn-standalone is deprecated, but still supported
-    if ((master == "yarn-cluster" || master == "yarn-standalone") &&
```

Comment:
FWIW I don't think `yarn-standalone` has existed for a long time and can be ignored, yes.